src/js/select2/dropdown/search.js

```diff
@@ -9,9 +9,9 @@ define([
 
     var $search = $(
       '<span class="select2-search select2-search--dropdown">' +
-        '<input class="select2-search__field" type="search" tabindex="-1"' +
+        '<input class="select2-search__field" type="text" tabindex="-1"' +
```

Comment:
I assume this change is necessary for your changes, but is `search` not more semantic?